inventory/install-config-example.yml

```diff
@@ -0,0 +1,30 @@
+---
+baseDomain: example.com
+machines:
+- name: master
+  replicas: 1
+- name: worker
+  # This should always be zero for openshift-ansible
+  replicas: 0
```

Comment:
@michaelgugino curious, why this will always be 0 ? isn't the installer creating the workers here or we decided for a different path for libvirt ?

Reply:
_sorry i didn't have time to study the whole code base in this branch hence i might ask a silly question_

Reply:
Yes, as you probably figured out, this value is merely used for machinesets, of which we don't want any replicas in the cluster deployed from openshift-ansible.   The context is the user 'brings their own hosts'